README.md

```diff
@@ -0,0 +1,149 @@
+# Automated K6 Load Testing with Virtual Users
+
+This repository provides a K6 test script designed to automate load testing for HTTP-based systems. It allows you to simulate multiple virtual users (VUs), gradually ramp up the load, and evaluate system performance based on defined thresholds.
```

Comment:
niet alleen gradually ramp up the load. het vind de proximate max VU's die iedere seconde elke endpoint aanspreken.